tests/test_processing_common.py

```diff
@@ -86,9 +86,16 @@ def get_processor(self):
         processor = self.processor_class(**components, **self.prepare_processor_dict())
         return processor
 
+    def prepare_text_inputs(self, batch_size=None):
+        if batch_size is not None and batch_size > 1:
+            return ["lower newer", "upper older longer string"] + ["lower newer"] * (batch_size - 2)
```

Comment:
This is a bit funny - what if I want a batch size of 1? This should also be possible as the processors should be able to handle both `"lower newer"` and `["lower newer"]` as input